plugins/cgroups/slurm/src/attr.rs

```diff
@@ -80,4 +85,28 @@ mod tests {
         let attrs: Vec<(String, AttributeValue)> = vec![];
         assert_eq!(find_jobid_in_attrs(&attrs), None);
     }
+
+    #[test]
+    fn test_find_jobstep_in_attrs() {
+        let attrs: Vec<(String, AttributeValue)> = vec![
+            ("job_step".to_string(), AttributeValue::String("1910".to_string())),
+            ("Saphira".to_string(), AttributeValue::String("Eragon".to_string())),
+        ];
+        assert_eq!(find_job_step_in_attrs(&attrs), Some("1910".to_string()));
+    }
+
+    #[test]
+    fn test_find_job_step_in_attrs_not_existing() {
+        let attrs: Vec<(String, AttributeValue)> = vec![
+            ("not_job_step".to_string(), AttributeValue::String("1512".to_string())),
+            ("Glaedr".to_string(), AttributeValue::String("Oromis".to_string())),
+        ];
+        assert_eq!(find_job_step_in_attrs(&attrs), None);
+    }
+
+    #[test]
+    fn test_find_job_step_in_empty_vec() {
+        let attrs: Vec<(String, AttributeValue)> = vec![];
+        assert_eq!(find_job_step_in_attrs(&attrs), None);
+    }
```

Comment:
Could you add tests for the JobTagger's attributes_for_cgroup, with several levels: job, task, step?